docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,490 @@
+          "AvgTicketPrice" : 816.1058877497752,
+          "Carrier" : "Kibana Airlines",
+          "FlightDelayMin" : 0,
+          "OriginRegion" : "CO-CUN",
+          "FlightDelayType" : "No Delay",
+          "DestAirportID" : "NRT",
+          "timestamp" : "2019-11-12T17:06:35",
+          "Dest" : "Narita International Airport",
+          "FlightTimeHour" : 13.210330384036373,
+          "Cancelled" : false,
+          "DistanceKilometers" : 14267.156814759282,
+          "OriginCityName" : "Bogota",
+          "DestWeather" : "Sunny",
+          "OriginCountry" : "CO",
+          "ml__id_copy" : "-7OQg24B95sXiIrm0EZo",
+          "DestCountry" : "JP",
+          "DestRegion" : "SE-BD",
+          "OriginAirportID" : "BOG",
+          "DestCityName" : "Tokyo",
+          "ml" : {
+            "top_classes" : [ <2>
+              {
+                "class_probability" : 0.939335365058496, <3>
+                "class_name" : "false" <4>
+              },
+              {
+                "class_probability" : 0.06066463494150393, <5>
+                "class_name" : "true"
+              }
+            ],
+            "FlightDelay_prediction" : "false", <6>
+            "is_training" : false <7>
+          }
+----
+<1> The `dependent_variable` with the ground truth value. This is what we are 
+trying to predict with the {classanalysis}.
+<2> The top classes object contains the predicted classes with the highest 
+probability.
+<3> The probability that `FlightDelay` is false (the larger probability of the 
+two, hence this is the predicted class). This is a value between 0 and 1. The 
+higher the number the higher the probability that the datapoint belongs to the 
+class.
+<4> Name of the class.
+<5> The probability that `FlightDelay` is true.
+<6> The prediction. The field name is suffixed with `_prediction` by default. 
+You can specify the field name by defining `prediction_field_name` via the API. 
+<7> Indicates that this document was not used in the training set.
+
+The example above shows that the analysis has predicted the probability of all 
+possible classes. In this case, there are two classes: `true` and `false`. The 
+class names along with the probability of the given classes are displayed in the 
+`top_classes` object. The most probable class is the prediction. In the example 
+above, `false` has a `class_probability` of 0.93 while `true` has only 0.06, so 
+the prediction will be `false` which coincides with the ground truth contained 
+by the `FlightDelay` field. The class probability values help you understand how 
+sure the model is about the prediction. The higher number means that the model 
+is more confident.
+
+If a document doesn't contain a prediction field, then it is excluded from the 
+analysis. In order to be analyzed, a document must contain at least one field 
+with a supported data type (`numeric`, `boolean`, `text`, `keyword` or `ip`) and 
+must not contain arrays with more than one item.
```

Comment:
I am a bit torn about what to do here, because in my opinion having different exclude behaviour for a document with one element vs doc with multiple elements in the same field is a bit confusing and I'd like to add a clarifying point for this somewhere in the docs. It might not necessarily be here in this example, though. 
I need to think about this a bit and will ping @dimitris-athanasiou for a further conversation.